src/component/CompressedRecipe/QuickSummaryItems/CostSummary/CostSummary.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import styles from '../QuickSummary.module.scss';
+
+const CostSummary = (props) => {
+  return (
+    <figure className={styles.QuickSummaryContainer}>
+      <img
+        className={styles.QuickSummaryIcon}
+        src="https://spoonacular.com/application/frontend/images/badges/cheap.svg"
+        alt="cost"
+      ></img>
+      <figcaption>${props.pricePerServing}</figcaption>
+    </figure>
```

Comment:
still using wrong html tags. Figure, figcaption does not seem correct on this place

Reply:
i just used this as a reference https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_figcaption

Reply:
changes made